CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## NEXT VERSION
 
+## 20231204 (latest)
+
+* **GCC version: 9.5.0**
 * devel: add tini (init system)
 * devel: add net-tools
+* phoenix-rtos-project commit `5c6d72a`
```

Comment:
Please note that when you are re-building the `devel` image hostutils like psu and phoenixd are built using the newest project version, not this stored in `/opt/phoenix/git-version`. However you can make sure that it didn't use files from some older image (you can check it by `ls -la` on `phoenixd` within this already built `devel` image)

```suggestion
* phoenix-rtos-project commit
  * devel image: `xxx`
  * build image: `xxx`
```